cmd/ctr/commands/containers/containers.go

```diff
@@ -83,7 +83,7 @@ var createCommand = cli.Command{
 			return err
 		}
 		defer cancel()
-		_, err = run.NewContainer(ctx, client, context)
+		_, err = run.NewContainer(ctx, client, context, "")
```

Comment:
Do we also make `ctr container create` support `--sandbox` as well? That would be more consistent

Reply:
Probably best to, can add. Also your other suggestions make this simple